kb-footer/src/kb-footer-column.ts

```diff
@@ -2,73 +2,49 @@ import {directive} from "lit-html/directive.js";
 import {html, TemplateResult} from "lit";
 import {AsyncDirective} from "lit-html/async-directive.js";
 import type {Part, DirectiveParameters} from 'lit/directive.js';
-import "@kb-dk/kb-icon/index.js";
-import {defaultsColumns} from './default_column_data.js';
+import "@kb-dk/kb-icon";
+import {defaultsColumnsDA, defaultsColumnsEN, lastColumnDA, lastColumnEN} from './default_column_data.js';
 
 class KbFooterColumn extends AsyncDirective {
 
-    update = (part:Part, [column]: DirectiveParameters<this>): void => {
-        let footerColumnDataUrl: string = `/jsonapi/node/site/e065d5e7-a348-4384-9859-c17841d03019?fields[node--site]=footer_column_${column}`;
-        this.fetchData(footerColumnDataUrl, column);
+    update = (part:Part, [column, language]: DirectiveParameters<this>): void => {
+        let footerColumnDataUrl: string = `/footerapi${language === 'en' ? "/en" : ""}/jsonapi/node/site/e065d5e7-a348-4384-9859-c17841d03019?fields[node--site]=footer_column_${column}`;
+        this.fetchData(footerColumnDataUrl, column, language);
     };
 
-    fetchData = (url: string, column: number): void => {
+    fetchData = (url: string, column: number, language: string): void => {
         fetch(url)
```

Comment:
I know it is implicit when working with web components but I still like the use of the native fetch API for XHR:)